frontend/components/card/discussion/CardDiscussionInput.vue

```diff
@@ -127,47 +128,38 @@
     </div>
   </div>
 </template>
-
 <script setup lang="ts">
 import type { DiscussionInput } from "~/types/card-discussion-input";
-
+import TooltipDiscussionWarning from '@/components/tooltip/TooltipDiscussionWarning.vue';
```

Comment:
Also feel free to remove this line. Nuxt loads all the components automatically 😊 We do need to load the types, but not components ✨

Reply:
Once I remove this line the feature doesn't work anymore

Reply:
This is definitely unexpected 🤔 Are you building with Docker? Should be fine regardless though. Notice that we're not loading components throughout the codebase, so what might be happening is there's an error in the component itself.

Reply:
Yes, I'm building with Docker